Gemfile.lock

```diff
@@ -112,3 +112,6 @@ DEPENDENCIES
   rails (= 3.0.9)
   rb-fsevent (>= 0.9.2)
   rspec-rails (~> 2.11)
+
+BUNDLED WITH
+   1.17.3
```

Comment:
This (old) gem specifies a development dependence of Rails 3.x, which isn't compatible with bundle 2.x

I didn't want to take on the scope creep of fixing that too...